smithy-openapi/src/test/java/software/amazon/smithy/openapi/fromsmithy/ServiceLevelErrorOverrideTest.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package software.amazon.smithy.openapi.fromsmithy;
+
+import org.junit.jupiter.api.Test;
+import software.amazon.smithy.model.Model;
+import software.amazon.smithy.model.node.Node;
+import software.amazon.smithy.model.node.ObjectNode;
+import software.amazon.smithy.model.shapes.ShapeId;
+import software.amazon.smithy.openapi.OpenApiConfig;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Tests that service-level errors don't override operation-specific error examples.
+ * This reproduces the healthcare service issue where ValidationException (service-level)
+ * overrides EligibilityCheckOutputValidationErrors (operation-level) examples.
+ */
+public class ServiceLevelErrorOverrideTest {
```

Comment:
Rename this to something like `ErrorDeconflictingExamplesTest`, and get rid of Stedi-specific jargon, like `EligibilityCheckOutputValidationErrors`, `healthcare service`, etc.